trace/topic.go

```diff
@@ -153,9 +153,10 @@ type (
 
 	// Internals: https://github.com/ydb-platform/ydb-go-sdk/blob/master/VERSIONING.md#internals
 	TopicReaderStartInfo struct {
-		ReaderID int64
-		Consumer string
-		Error    error
+		LogContext *context.Context
```

Comment:
What abount simple Context instead of LogContext?

Reply:
I would prefer to keep it as-is to mark what the context is for